README.md

```diff
@@ -0,0 +1,59 @@
+# Q Applet: Flight Price Watcher
+
+This applet will help you find the best flight prices.
+
+[GitHub repository](https://github.com/daskeyboard/daskeyboard-applet--fly-me)
+
+## Example
+
+Displays a notification on Das Keyboard Q series according to the price of the flight.
+First, the user has to set a threshold for his flight. In order to do it he has to go on the skyscanner website and
```

Comment:
Please do not use threshold. Explain instead what the threshold is. Remove all references to Threshold please